CONTRIBUTING.md

```diff
@@ -16,6 +16,22 @@ Let us know if these look good to you.
 You can use `npm test` to validate data against the schema. You might need to install the devDependencies using `npm install --only=dev`.
 The JSON data is validated against the schema using [`ajv`](http://epoberezkin.github.io/ajv/).
 
+### Optional: Validate/cross-reference against web API confluence dashboard
+If the feature you're interested is a JavaScript API you can cross-reference data against the [Web API Confluence Dashboard](https://web-confluence.appspot.com/) using the `confluence` npm script. This script will overwrite data in your current working tree according to data from the dashboard.
+
+Examples:
+
+```shell
+# Load confluence data for ServiceWorker
+npm run confluence -- --interfaces=ServiceWorker
+
+# Fill in missing/ambiguous Firefox data on known interfaces
+npm run confluence -- --browsers=firefox --fill-only
+
+# Print documentation on full list of options
+npm run confluence -- --help
+```
+
```

Comment:
There are no typos, but you should add some boilerplate about the limitations of Confluence.

Reply:
Any thoughts on this?

Reply:
Happy to add Confluence as an optional tool, but I agree, it would be nice to add a note here saying that Confluence shouldn't always be trusted as The Truth. The recent Confluence-powered PRs had a few things that made no sense and especially if `version_removed` is added by Confluence, contributors should be suspicious.

Reply:
Added caveat about Confluence fallibility. @Elchi3 WDYT?